trellis/ssh-keys.md

```diff
@@ -124,8 +125,16 @@ users:
 
 The example above also demonstrates the option of creating `another_user` whose primary group is `some_group`, but who is also in `some_other_group`, and who has public SSH keys for `swalkinshaw`.
 
+## Removing keys
+
+Removing a key and re-provisioning the server will remove the key in the `authorized_keys` file as of [#1576](https://github.com/roots/trellis/pull/1576).
```

Comment:
I think this could be clearer. How about this?

```suggestion
Removing a key from the configuration and re-provisioning the server does not remove the key from the server's `authorized_keys` file by default. To reset SSH keys, Trellis supports an opt-in "single use" `reset_user_ssh_keys` variable (as of [#1576](https://github.com/roots/trellis/pull/1576)):

Reply:
I like it. Thanks! 🙌